evap/evaluation/views.py

```diff
@@ -94,3 +101,26 @@ def faq(request):
 
 def legal_notice(request):
     return render(request, "legal_notice.html", dict())
+
+
+@require_POST
+def feedback_send(request):
+    sender_email = request.POST.get("sender_email")
```

Comment:
this is optional, right?

if the user doesn't provide one, django uses `DEFAULT_FROM_EMAIL`. maybe we should add a setting like `DEFAULT_FEEDBACK_EMAIL = 'feedback@localhost'` and use that if the user doesn't enter an address?